examples/Cargo.toml

```diff
@@ -7,6 +7,7 @@ authors = ["Sebastian Dröge <sebastian@centricular.com>"]
 glib = { version = "0.1.3", git = "https://github.com/gtk-rs/glib" }
 gstreamer = { path = "../gstreamer" }
 gstreamer-app = { path = "../gstreamer-app" }
+gstreamer-player = { path = "../gstreamer-player", features = ["v1_12"] }
```

Comment:
You need to define a feature for the examples, that then depends on gstreamer-player. So that the player example is only built if you do `cargo build --features player` for example.

Otherwise we force everybody wanting to compile the examples to use 1.10 (the minimum version needed otherwise right now is 1.8)